### DISCUSSION TITLE
Plugins?

### DISCUSSION THREAD

**Discussion by nirmalhk7** [mrgnr/camus#41] [Ideas]
Hi, 
Well done with the work so far. I''m interested in making this connect to a Slack channel of my choice, which brings me to a broader suggestion: to make a plugins facility, so that the community can make plugins as they'd wish

**Comment by mrgnr** [comment]
Hi @nirmalhk7, thanks for your suggestion. I'd definitely like to add (optional/configurable) integration with third-party services in the future. Could you elaborate a bit on what kinds of plugins you'd like to see (e.g. the ability to add user interface widgets, or ...)? By connecting to a Slack channel, do you mean the ability to send audio/video/text from Camus to Slack, or something like a Slack bot that allows you to create a Camus room from Slack?
